indico/htdocs/js/indico/modules/abstracts.js

```diff
@@ -15,7 +15,8 @@
  * along with Indico; if not, see <http://www.gnu.org/licenses/>.
  */
 
-/* global setupListGenerator:false, setupSearchBox:false, applySearchFilters:false, Palette:false */
+/* global setupListGenerator:false, setupSearchBox:false, applySearchFilters:false, Palette:false,
+          avatarPalette: false */
```

Comment:
Does this actually work? The last time I tried wrapping the line it did not recognize the ones from the new lines so I had to repeat the `/* global ` part